content/en/agent/kubernetes/control_plane.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Kubernetes Control Plane Monitoring
+kind: documentation
+further_reading:
+    - link: 'agent/kubernetes/log'
+      tag: 'Documentation'
+      text: 'Collect your application logs'
+    - link: '/agent/kubernetes/apm'
+      tag: 'Documentation'
+      text: 'Collect your application traces'
+    - link: '/agent/kubernetes/prometheus'
+      tag: 'Documentation'
+      text: 'Collect your Prometheus metrics'
+    - link: '/agent/kubernetes/integrations'
+      tag: 'Documentation'
+      text: 'Collect automatically your application metrics and logs'
+    - link: '/agent/guide/autodiscovery-management'
+      tag: 'Documentation'
+      text: 'Limit data collection to a subset of containers only'
+    - link: '/agent/kubernetes/tag'
+      tag: 'Documentation'
+      text: 'Assign tags to all data emitted by a container'
+---
+
+## Overview
+
+This section aims to document specificities and to provide good base configurations for monitoring the Kubernetes Control Plane. You can then customize these configurations to add any Datadog feature.
+
+With four Datadog integrations for the [API Server][1], [ETCD][2], [Controller Manager][3], and [Scheduler][4], you can collect key metrics from all four components of the Kubernetes Control Plane.
```

Comment:
This is subjective, so no worries if you prefer it as is - I would change `With four Datadog integrations for the...` -> `With Datadog integrations for the...` since you say `from all four components` later in the sentence